### ISSUE TITLE
Remove pipenv dependency from development

### ISSUE THREAD

**Issue by fluffy-critter** [PlaidWeb/Publ#364]
## Expected Behavior

Instead of using a `Pipfile`/`Pipfile.lock` for local dev environment it'd be better for the Makefile to set up a local virtualenv for running the tools from, or something.

In the long run maybe we should switch to `poetry` instead of `setuptools` but that has its own set of issues that seem tangential to what we care about with Publ.

**Comment by fluffy-critter** [comment]
Downside: this complicates Windows-based development, since `virtualenv` uses a different directory convention there (`env/scripts` instead of `env/bin`).

It also seems like the issues with `pipenv` are resolving themselves; it's Under New Management™ and they're working on unblocking their release pipeline. So this might become a wontfix.

**Comment by fluffy-critter** [comment]
yeah let's not do this after all
